external/.gitignore

```diff
@@ -0,0 +1,2 @@
+build/
+.vscode/.cmaketools.json
```

Comment:
This isn't related to this PR's intent and shouldn't be here. In general it's better to just ignore the entire `.vscode` directory at the top-level project root. For this PR though there is no need to add this. Especially to the `external/` folder when it can just be added at the top-level.

Reply:
Fixed in [bbc8649](https://github.com/Wunkolo/vkblam/pull/1/commits/bbc8649cf4c93550cbd1a16611d7e3a3c108c38b). Closing.